src/cmds/proc/thread.c

```diff
@@ -29,22 +30,24 @@ static void print_stat(void) {
 	int total;
 	struct task *task;
 
-	printf(" %4s %4s %8s %7s %10s\n", "id", "tid", "priority", "state", "time");
+	printf(" %4s %16s %8s %7s %10s %12s\n", "id", "tid/Task name", "priority", "state", "time", "Stack size");
 
 	running = sleeping = suspended = 0;
 
 	sched_lock();
 	{
 		task_foreach(task) {
 			task_foreach_thread(t, task) {
-				printf(" %4d %4d %8d %c %c %c %c %9lds\n",
+				printf(" %4d %4d/%-11s %8d %c %c %c %c %9lds %9d\n",
```

Comment:
One of the templates aarch64 failed - https://travis-ci.org/embox/embox/jobs/659123344#L1749

It's probably because you should PRIu32 or maybe "%zu" for this instead of `%d`. It's because of thread_stack_get_size returns size_t

Reply:
You can find more about PRIu32 with grep in src/

Reply:
Okay, I am checking it.

Reply:
Hey @alexkalmuk when I was checking it, I found that my offline environment is showing 'size_t' to be 'unsigned int'. And in https://travis-ci.org/embox/embox/jobs/659123344#L1749 its showing 'size_t' to be 'long unsigned int' .
So what should I do?

Reply:
![Screenshot from 2020-03-06 19-56-39](https://user-images.githubusercontent.com/45538418/76091983-a1cd7400-5fe4-11ea-802e-5533a6852290.png)

Reply:
Can you please try "%u" instead of "%d"?

Reply:
I will try to reproduce it so

Reply:
But please try usual "%u" first, it didn't help?

Reply:
Ok, I verified that `%zu` works fine, can you do that please?

Reply:
Yeah, I'll do that.
Just give me 2 mins.

Reply:
It's not a problem, I just proposed the fix, you are free to that when you want :)

Reply:
Okay

Reply:
Btw can you please suggest me some other issue that requires some debugging using gdb and is easy or intermediate level, because I tried to use gdb and I wasn't much comfortable with that.